packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -376,8 +376,16 @@ function useRestoreFocus(scopeRef: RefObject<HTMLElement[]>, restoreFocus: boole
         if (nextElement) {
           focusElement(nextElement, true);
         } else {
-          // If there is no next element, blur the focused element to move focus to the body.
-          focusedElement.blur();
+          if (contain) {
```

Comment:
Isn't this backwards? If it's contained, then we _don't_ want focus to go back to the body, right?